lib/ueberauth.ex

```diff
@@ -69,7 +69,7 @@ defmodule Ueberauth do
   In phoenix setup a pipeline:
 
       pipeline :ueberauth do
-        Ueberauth.plug "/auth"
+        plug Ueberauth, base_path: "/auth"
```

Comment:
keep in mind I'm murky on how this all works on the whole and when the plug should be invoked. Find me on irc if what I'm saying is making no sense :)